dtrace/lib/common.d

```diff
@@ -2,14 +2,14 @@
 #pragma D depends_on provider ip
 
 typedef struct flow_id_sdt_arg {
-	int		af;
-	ipaddr_t	src_ip4;
-	ipaddr_t	dst_ip4;
-	in6_addr_t	src_ip6;
-	in6_addr_t	dst_ip6;
+	uint8_t		proto;
+	uint16_t		af;
```

Comment:
Nit: is Github messed up alignment here, or is there an extra tab?

Reply:
Yup, done.